int16 left_analog
int16 right_analog
int16 r1_button
int16 r2_button
